File: SemantikosClientesWS/ClienteJava/src/cl/minsal/semantikos/ws/shared/CrossMapsIndirectosPorIDDescripcion.java

package cl.minsal.semantikos.ws.shared;

import javax.xml.bind.annotation.XmlAccessType;
import javax.xml.bind.annotation.XmlAccessorType;
import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlType;


/**
 * <p>Java class for crossMapsIndirectosPorIDDescripcion complex type.
 * 
 * <p>The following schema fragment specifies the expected content contained within this class.
 * 
 * <pre>
 * &lt;complexType name="crossMapsIndirectosPorIDDescripcion">
 *   &lt;complexContent>
 *     &lt;restriction base="{http://www.w3.org/2001/XMLSchema}anyType">
 *       &lt;sequence>
 *         &lt;element name="DescripcionID" type="{http://www.w3.org/2001/XMLSchema}string"/>
 *       &lt;/sequence>
 *     &lt;/restriction>
 *   &lt;/complexContent>
 * &lt;/complexType>
 * </pre>
 * 
 * 
 */
@XmlAccessorType(XmlAccessType.FIELD)
@XmlType(name = "crossMapsIndirectosPorIDDescripcion", propOrder = {
    "descripcionID"
})
public class CrossMapsIndirectosPorIDDescripcion {

    @XmlElement(name = "DescripcionID", required = true)
    protected String descripcionID;

    /**
     * Gets the value of the descripcionID property.
     * 
     * @return
     *     possible object is
     *     {@link String }
     *     
     */
    public String getDescripcionID() {
        return descripcionID;
    }

    /**
     * Sets the value of the descripcionID property.
     * 
     * @param value
     *     allowed object is
     *     {@link String }
     *     
     */
    public void setDescripcionID(String value) {
        this.descripcionID = value;
    }

}

File: SemantikosClientesWS/clientePHP/src/Semantikos/ClientBundle/API/crossMapsIndirectosPorIDDescripcion.php
<?php

namespace Semantikos\ClientBundle\API;

class crossMapsIndirectosPorIDDescripcion
{

    /**
     * @var string $DescripcionID
     */
    protected $DescripcionID = null;

    /**
     * @param string $DescripcionID
     */
    public function __construct($DescripcionID)
    {
      $this->DescripcionID = $DescripcionID;
    }

    /**
     * @return string
     */
    public function getDescripcionID()
    {
      return $this->DescripcionID;
    }

    /**
     * @param string $DescripcionID
     * @return crossMapsIndirectosPorIDDescripcion
     */
    public function setDescripcionID($DescripcionID)
    {
      $this->DescripcionID = $DescripcionID;
      return $this;
    }

}
